lib/runtime/helpers_unchecked_extrinsic_test.go

```diff
@@ -0,0 +1,228 @@
+package runtime
+
+import (
+	"fmt"
+	"math/big"
+	"testing"
+
+	"github.com/ChainSafe/gossamer/dot/types"
+	"github.com/ChainSafe/gossamer/lib/common"
+	"github.com/ChainSafe/gossamer/lib/runtime/extrinsic"
+	"github.com/stretchr/testify/require"
+)
+
+func TestApplyExtrinsic_AuthoritiesChange_UncheckedExt(t *testing.T) {
```

Comment:
I think this can be removed, `AuthoritiesChange` exts aren't relevant to the new runtime

Reply:
Nice, I was having a hard time trying to figure out which Pallet this should use.